diem-node/src/lib.rs

```diff
@@ -415,6 +435,32 @@ pub fn setup_environment(node_config: &NodeConfig, logger: Option<Arc<Logger>>)
     // TODO set up on-chain discovery network based on UpstreamConfig.fallback_network
     // and pass network handles to mempool/state sync
 
+    // For now, spawn all of the storage-service servers on the same runtime.
+    let storage_service_rt = Builder::new_multi_thread()
```

Comment:
nit: do you think we could put this logic into its own helper function? It might be cleaner to just have a single call to something like:
```
let storage_service_rt = spawn_storage_service(...);
```

which would internally spawn the server and client tasks and wire up all the network handlers, etc.